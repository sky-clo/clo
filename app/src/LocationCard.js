import React, { Component } from "react";

class LocationCard extends Component {
  render() {
    return (
      <th>
        <article class="c-tile c-tile--square has-focus">
          <a class="c-tile__link c-shine-context" href="#">
            <div class="c-tile__content">
              <div class="c-tile__media">
                <img
                  alt="Example image"
                  class="c-tile__poster"
                  src="https://www.sky.com/assets/toolkit/docs/tile/example.jpg"
                />
              </div>
              <div class="c-tile__body c-tile__caption">
                <p class="c-tile__title c-heading-delta">
                  {this.props.location.locationTitle}
                </p>
              </div>
            </div>
          </a>
        </article>
      </th>
    );
  }
}

export default LocationCard;
