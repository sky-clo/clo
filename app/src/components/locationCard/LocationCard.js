import React, { Component } from "react";

class LocationCard extends Component {
  render() {
    return (
      <div className="o-layout__item">
        <div className="c-tile c-tile--square">
          <a className="c-tile__link c-shine-context" href="#">
            <div className="c-tile__content">
              <div className="c-tile__media">
                <img
                  alt="Example image"
                  className="c-tile__poster"
                  src={this.props.img_url}
                />
              </div>
              <div className="c-tile__body c-tile__caption">
                <p className="c-tile__title c-heading-delta">
                  {this.props.name}
                </p>
              </div>
            </div>
          </a>
        </div>
      </div>
    );
  }
}

export default LocationCard;
