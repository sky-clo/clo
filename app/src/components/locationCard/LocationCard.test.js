import React from "react";
import { shallow } from "enzyme";

import LocationCard from "./LocationCard";

describe("LocationCard", () => {
  it("should match snapshot", () => {
    const render = shallow(
      <LocationCard
        name="London"
        imgSrc="https://static.skyassets.com/contentstack/assets/bltdc2476c7b6b194dd/blt556529d1b8bf298d/5edfe30f98e62f38ec84dadd/2020_Sky_Logo.png"
      />
    );
    expect(render).toMatchSnapshot();
  });
});
