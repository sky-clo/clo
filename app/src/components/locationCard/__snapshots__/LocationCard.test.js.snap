// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationCard should match snapshot 1`] = `
<div
  className="o-layout__item"
>
  <div
    className="c-tile c-tile--square"
  >
    <a
      className="c-tile__link c-shine-context"
      href="/location/london"
    >
      <div
        className="c-tile__content"
      >
        <div
          className="c-tile__media"
        >
          <img
            alt="London"
            className="c-tile__poster"
            src="https://static.skyassets.com/contentstack/assets/bltdc2476c7b6b194dd/blt556529d1b8bf298d/5edfe30f98e62f38ec84dadd/2020_Sky_Logo.png"
          />
        </div>
        <div
          className="c-tile__body c-tile__caption"
        >
          <p
            className="c-tile__title c-heading-delta"
          >
            London
          </p>
        </div>
      </div>
    </a>
  </div>
</div>
`;
